deps.edn

```diff
@@ -73,6 +73,10 @@
                                                                  :exclusions [org.babashka/sci]}
                                io.github.nextjournal/clerk-slideshow {:git/sha "11a83fea564da04b9d17734f2031a4921d917893"}}}
 
+           :nextjournal/garden {:exec-fn nextjournal.clerk/serve!
+                                :exec-args {:index "book.clj"}
+                                :nextjournal.garden/aliases [:demo]}
+
```

Comment:
I can [move this to its own deps.edn in a subdirectory](https://nextjournal.slack.com/archives/C047QGK2UG7/p1697119000296089) if we don't want it here.